cmd/kops/get.go

```diff
@@ -125,8 +125,7 @@ func RunGet(context Factory, out io.Writer, options *GetOptions) error {
 	}
 
 	if cluster == nil {
-		fmt.Fprintf(os.Stderr, "No cluster found\n")
-		return nil
+		return fmt.Errorf("No cluster found")
```

Comment:
We can do another PR, but I think we should be printing out the name, if we have one